apps/studio/src/vendor/sql-hint/index.js

```diff
@@ -231,22 +257,42 @@
         var lineText = query[i];
         eachWord(lineText, function(word) {
           var wordUpperCase = word.toUpperCase();
-          if (wordUpperCase === aliasUpperCase && getTable(previousWord))
+          if (wordUpperCase === aliasUpperCase && (tableWord = getTable(previousWord)))
```

Comment:
If possible I would prefer if we didn't assign within conditionals. I'm also not sure that `table` is entirely necessary anymore, but I haven't dug around in this area of the code.

Reply:
I assigned within conditionals because I thought it could minimize the lines of changes. But I'll fix that if it's preferred no to.

Reply:
I personally just think it makes the code a bit harder to read.